layouts/shortcodes/ecosystem/registry/search-form.html

```diff
@@ -34,6 +34,39 @@
 {{ end -}}
 {{ $types = $types | uniq | sort -}}
 
+{{ $counter := 0 -}}
+{{ $entries := slice -}}
+{{ range $key, $entry := .Site.Data.registry -}}
+  {{ $flags := slice -}}
+  {{ if .isNative -}}
+    {{ $flags = $flags | append "native" -}}
+  {{ end -}}
+  {{ if .isFirstParty -}}
+    {{ $flags = $flags | append "first_party" -}}
+  {{ end -}}
+  {{ if .isNew -}}
+    {{ $flags = $flags | append "new" -}}
+  {{ end -}}
+  {{ if .usedInDemo -}}
+    {{ $flags = $flags | append "used in demo" -}}
```

Comment:
```suggestion
    {{ $flags = $flags | append "used_in_demo" -}}
```

Although currently not used, here you also might want to use _ instead of spaces